corehq/apps/userreports/reports/data_source.py

```diff
@@ -106,6 +107,8 @@ def column_warnings(self):
 
     @memoized
     def get_data(self, slugs=None):
+        if len(self.columns) > 50:
+            raise UserReportsError(_("This report has too many columns to be displayed"))
```

Comment:
should we consider giving the user some clearer way to resolve this? (going to merge now since this is still behind a flag but ping @amsagoff )